common/firestore/MessageWithId.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.common.firestore;
+
+import com.google.auto.value.AutoValue;
+import com.google.protobuf.Message;
+
+@AutoValue
+public abstract class MessageWithId {
+  // A holder class for a proto message and an ID, typically the Firestore document ID.
```

Comment:
This comment should be for the whole class.

Reply:
@oferb ae17bcd6192338e64122e84142d2a2c4c2d42604